README.md

```diff
@@ -127,12 +127,15 @@ Note that you can define more than one dashboard under the `dashboardTemplates`
 
 #### Configurations
 
-In the `configuration` section of the `definition.yml` file you can tweak the entity's behavior. At the moment you can only configure the entity's expiration time. 
+In the `configuration` section of the `definition.yml` file you can tweak the entity's behavior. 
+At the moment we accept two optional configuration parameters `entityExpirationTime` and `alertable`.
 
 ```yaml
 configuration:
   # The amount of time without receiving telemetry before an entity is deleted. Defaults to EIGHT_DAYS
   entityExpirationTime: EIGHT_DAYS
+  # Defines whether the entities of this type should have an alert status associated, defaults to true
```

Comment:
We are actually defaulting to false right now, but almost all entities are `alertable`=true  and for synthesis true probably also makes more sense, wdyt?

Reply:
It may be worth adding a brief explanation about how this field will be populated. E.g. "an alertable entity's metadata will include a field `alertSeverity` that is updated when the telemetry associated to this entity breaks an alerting condition."

Reply:
ping! (I can help out if you're busy!)